src/psyclone/dynamo0p3.py

```diff
@@ -808,7 +454,7 @@ def __init__(self, ast, name=None):
         # parse the arg_type metadata
         self._arg_descriptors = []
         for arg_type in self._inits:
-            self._arg_descriptors.append(DynArgDescriptor03(arg_type))
+            self._arg_descriptors.append(LFRicArgDescriptor(arg_type))
```

Comment:
Not your fault but since we're in this routine would you mind removing the import of `getkerneldescriptors` as pylint points out it has already been imported at line 52.

Reply:
Done.